cli/command/container/opts.go

```diff
@@ -236,9 +236,12 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.Int64Var(&copts.cpuPeriod, "cpu-period", 0, "Limit CPU CFS (Completely Fair Scheduler) period")
 	flags.Int64Var(&copts.cpuQuota, "cpu-quota", 0, "Limit CPU CFS (Completely Fair Scheduler) quota")
 	flags.Int64Var(&copts.cpuRealtimePeriod, "cpu-rt-period", 0, "Limit CPU real-time period in microseconds")
+	flags.SetAnnotation("cpu-rt-period", "version", []string{"1.25"})
```

Comment:
Yes I considered putting all the annotations at the end (looking a bit cleaner), but decided keeping them together with the flag also has it's advantages.

Ideally, something like this;

```go
flags.Var(&copts.foo, "foo", "Set foo option", flagAnnotation{"version", "1.25"},  flagAnnotation{"platform", "windows"})
```

or

```go
flags.Var(&copts.foo, "foo", "Set foo option").Annotate("version", "1.25").Annotate("platform", "windows")
```